### ISSUE TITLE
Question on idiom used in PowerSimulations

### ISSUE THREAD

**Issue by daniel-thom** [NREL/PowerSimulations.jl#217]
PowerSimulations frequently uses code like this:

```
var = 999.0*(PSY.get_activepower(g) > 0)
```

I'm not used to seeing multiplication between floats and bools and was confused the first time I saw it.  I would have written like this:

```
var = PSY.get_activepower(g) > 0 ? 999.0 : 0.0
```

@jd-lara @kdheepak  Is the former idiomatic for Julia developers?

**Comment by jd-lara** [comment]
This is a short cut, ugly one. The value that has to go into JuMP has to be 0.0 or 999.0 depending on the status of the unit

**Comment by kdheepak** [comment]
Use the latter.

**Comment by daniel-thom** [comment]
The action is now to find all places where this exists and change them.

**Comment by jd-lara** [comment]
closed via #218
